frontend/package.json

```diff
@@ -9,29 +9,32 @@
     "lint:fix": "vue-cli-service lint "
   },
   "dependencies": {
-    "axios": "0.18.1",
-    "vue": "2.6.10",
-    "vue-router": "3.0.2",
-    "vuelidate": "0.7.4",
-    "vuetify": "1.5.14",
-    "vuex": "3.0.1"
+    "axios": "^0.18.1",
+    "sass": "^1.22.0",
```

Comment:
as I understand it replaced `node-sass`? Why not in `devDep...`?

Reply:
Maybe here's the reason why scss stopped overriding vuetify classess...

Reply:
I doubt that as it wasn't yet merged.

Reply:
And scss doesn't work only on this particular branch. It works fine on master

Reply:
I didn't anticipate, that you test branches before merging.

Reply:
It seems like either compliment or insult simultaneously :P

Reply:
vuetify from version 2.0 requires sass instead of node-sass, so i didn't see point to keep them both